testing/middleware/builder/builder.go

```diff
@@ -56,6 +57,8 @@ const (
 	GetPayloadMethodV2 = "engine_getPayloadV2"
 	// GetPayloadMethodV3 v3 request string for JSON-RPC.
 	GetPayloadMethodV3 = "engine_getPayloadV3"
+	// GetPayloadMethodV4 v3 request string for JSON-RPC.
```

Comment:
```suggestion
	// GetPayloadMethodV4 v4 request string for JSON-RPC.
```
Actually, can't it be just `GetPayloadMethodV4 request string for JSON-RPC`? Similarly for other versions